test/RpcTest.cpp

```diff
@@ -1,8 +1,10 @@
 #define TEST_NO_MAIN 1
 #include "RpcTest.hpp"
 #include "acutest.h"
-
 namespace Casper {
+const std::string pub_key = "0203368dd2192831578c585e008eaa4c40d0f871326b3a6e536fb02831682fb6f525";
+const  std::string privKeyPemFile = "test/data/matterficasper1_secret_key.pem";
```

Comment:
I would change the name of it to: please_provide_your_valid_secret_key.pem

Reply:
I would keep that name as it is (please take a look at README now)